src/components/WorkspaceContainer.js

```diff
@@ -1,16 +1,14 @@
-import { useContext } from 'react'
+import React, { useContext } from 'react'
```

Comment:
importing React is no longer required as of react v17

Reply:
Hmm...will need to tweak the eslint settings then.